client/src/components/Landing.jsx

```diff
@@ -1,84 +1,47 @@
 import React, { useEffect } from 'react';
+import Button from 'plaid-threads/Button';
 
-import { useUsers } from '../services';
+import { useUsers, useCurrentUser } from '../services';
+import Login from './Login';
 import Banner from './Banner';
 import UserList from './UserList';
+import AddUserForm from './AddUserForm';
+
+import { useBoolean } from '../hooks';
 
 const PLAID_ENV = process.env.REACT_APP_PLAID_ENV;
 
-export default function Landing({ users }) {
-  const { getUsers } = useUsers();
+export default function Landing({}) {
+  const { getUsers, usersById } = useUsers();
+  const { userState, setCurrentUser } = useCurrentUser();
+  const [isAdding, showForm, hideForm, toggleForm] = useBoolean(false);
 
   useEffect(() => {
     getUsers();
-  }, [getUsers]);
+  }, [getUsers, usersById]);
+
+  useEffect(() => {
+    if (userState.newUser != null) {
+      setCurrentUser(userState.newUser);
+    }
+  }, [getUsers, usersById]);
+
+  const buttons = (
+    <div className="loginBtns">
+      <Login />
+      <Button onClick={toggleForm} centered inline>
+        New User
+      </Button>
+    </div>
+  );
```

Comment:
this should just be inline in the `return`, extracting it here is not providing a useful abstraction, just an additional place to jump to in the file when trying to follow the logic.